tests/test_numbers.py

```diff
@@ -121,6 +121,39 @@ def test_group_separator(self):
         assert numbers.format_currency(101299.98, 'EUR', locale='en_US', group_separator=True, format_type='name') == u'101,299.98 euros'
         assert numbers.format_percent(251234.1234, locale='sv_SE', group_separator=True) == u'25\xa0123\xa0412\xa0%'
 
+    def test_compact(self):
```

Comment:
Just to make sure: is there a test here that sees how `decimal_quantization` interacts with this formatting?

Reply:
I don't think it needs one since the argument is not used with compact format types. If `compact_fraction_digits` is set, it is `True`, otherwise it is effectively `False` even if manually set.

Reply:
👍 That's what I thought; thanks for the clarification.